examples/nextjs/package-lock.json

```diff
@@ -18,7 +18,7 @@
     },
     "../..": {
       "name": "@primer/react",
-      "version": "35.23.0",
+      "version": "35.24.0",
```

Comment:
curious why this wasn't checked in already?

Reply:
I think this might be a downside of not having the project setup with workspaces as the changeset release action is not aware of this workspace when `@primer/react` is updated